style.css

```diff
@@ -426,7 +426,8 @@ textarea {
 }
 
 .button:hover {
-  background-color: #6070ff;
+  background-color: #01041b;
+  color: #f9f6f6;
```

Comment:
I think you did a great job here. However, there are still some things to improve. There are still errors with the contrast ratio as you can see [here](https://www.aditus.io/button-contrast-checker/raw-githack-com-2022-11-08-at-13-03-52-120?failing). Kindly fix this. :pray: :pray: 

![image](https://user-images.githubusercontent.com/98363075/200573941-f4ee46f4-28a4-4ae6-ac62-62ffe05e5e6a.png)

![image](https://user-images.githubusercontent.com/98363075/200574024-d762613b-782d-4ecc-84d6-d958ce36a3db.png)

![image](https://user-images.githubusercontent.com/98363075/200574071-74542542-4f26-452b-a183-ed799ae0787a.png)

Reply:
@Santiago220991 please the colors are from the figma design, should we still change it ?

Reply:
@quavo19 Yes for this project you can change the colors. Also please do not forget to add the accessibility summary in your PR. :pray: :pray: This is a project requirement. :+1: :smiley: :smiley: